profiles.md

```diff
@@ -0,0 +1,234 @@
+# Researcher profiles Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+This repository allows to manage the profile item linked to a particular eperson. The specific type of item 
+with which the profile is modeled depends on the dspace configuration (by default it is the Person type). If the particular dspace instance doesn't handle the "profile" 
+type then the whole repository is disabled.
+
+## Get all Profile
+**GET /api/eperson/profiles/**
+
+This operation is not currently supported.
+
+## Single Profile
+**GET /api/eperson/profiles/<:eperson-uuid>**
+
+Provide information about a specific profile. Only the Administrators or the owner eperson can access this endpoint.
+
+The JSON response document is as follow
+
+```json
+{
+  "id": "eb645ef8-1373-41eb-bf67-6afcea7e2069",
+  "visible": true,
+  "type": "profile"
+}
+```
+
+Attributes:
+* id: the UUID of the EPerson who owns the profile. This also corresponds to the ID of the profile.
+* visible: indicates whether the profile is public (visible: true) or private (otherwise). If private only 
+Administrators & the EPerson themselves can search/access the profile.
+* type: the resource type (profile)
+
+Exposed links:
+* item: the item with which the profile is modeled
+* eperson: the eperson related to the profile
+
+It would respond with:
+* 200 OK - Returning the profile if there's a match
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions to view the profile
+* 404 Not found - if the profile or the eperson doesn't exist
+
+## Linked entities
+
+### Item
+**GET /api/eperson/profiles/<:eperson-uuid>/item**
+
+Returns the item that model the profile of the eperson with the given uuid. The JSON response document is as follow
+
+```json
+{
+  "id": "d0601db5-5ceb-4f8d-8762-bd5b3b97845d",
+  "uuid": "d0601db5-5ceb-4f8d-8762-bd5b3b97845d",
+  "name": null,
+  "handle": "123456789/511",
+  "metadata": {
+    "dspace.object.owner": [{
```

Comment:
Will it be required to restrict edits to this field from all endpoints?
Wouldn't it also be better to use the UUID in the value, as the value can otherwise deviate from the EPerson name (if the latter is updated)

Reply:
No for us it is not necessary to restrict the edit on this metadata. Regarding the value, it contains the name of the eperson and the authority contains the uuid of the eperson itself.

Reply:
My questions here are primarily:
* how will it handle issues with `dspace.object.owner` not being valid
* how will it handle a change to the eperson's name, will this metadata value be updated as well. I see duplication of names which is hard to keep in sync

Reply:
- The dspace.object.owner can be set manually only with the administrative edit; this metadata can therefore be corrupted as can all the others. We don't think specific management is necessary.
- I don't think it's a relevant problem (it is a common situation for all relations via authority). If we really want to manage a possible update we can modify the logic that changes the name of the eperson to also update a possible connected profile.

Reply:
@LucaGiamminonni what I'm mainly pointing out here is that you're duplicating the name, and I don't see any reason for it. Unless you have a reason to duplicate the name, I would recommend to use the UUID in the value, which ensures there's no risk in the name being outdated

Reply:
@benbosman the eperson name must be on the metadata because it could be shown. There cannot be the id which instead must be used to set the authority.